src/server/common/helpers/audit/audit-message.js

```diff
@@ -0,0 +1,15 @@
+function auditMessage({ event, repository, user }) {
```

Comment:
feels a bit odd hardcoding repository as in you will have to add more objects in the future. Could this just be `data` and then the user of this method could send anything in on `data`?

Reply:
Could be data. The schema has it as optional. a lot of audit events will have repo and user and then an action. But from what we have at the moment 50% do not have repo. :/ Data seems a little abstract but maybe a compromise. Or yet another json inside json...